eZ/Publish/SPI/Specification/Tests/Content/ContentContainerSpecificationTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Specification\Tests\Content;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\Core\Search\Tests\TestCase;
```

Comment:
Seems like an accidental import. You've meant to add the he one from PHPUnit, right? ;)

Reply:
Right .. :)